src/trx-generator.ts

```diff
@@ -216,9 +216,11 @@ const renderTestSuiteResult = (
         );
       }
     });
-  } else if (testSuiteResult.failureMessage) {
+  }
+
+  if (testSuiteResult.failureMessage) {
```

Comment:
if other tests fail, those are getting included in the failureMessage too (even when there is not a top level failure)

Reply:
Fixed in 9c134bf, we should use `testExecError` instead for file-level failures